Dockerfile

```diff
@@ -10,23 +10,26 @@ ENV PATH="./node_modules/.bin:$PATH" \
     TZ=:/etc/localtime \
     LD_LIBRARY_PATH="/opt/oracle/instantclient/:$LD_LIBRARY_PATH" \
     ORACLE_HOME=/opt/oracle/instantclient/ \
+    # This is to fix 'error:0308010C:digital envelope routines::unsupported' in Node 18
+    # the proper fix should be upgrading webpack and babel-loader
+    NODE_OPTIONS='--openssl-legacy-provider' \
     DB=$DB
 
 USER root
 
-RUN dnf -y module enable ruby:2.7 nodejs:16 mysql:8.0 \
+RUN dnf -y module enable ruby:3.1 nodejs:18 mariadb:10.11 \
     && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs \
-        ruby-devel rubygem-rdoc rubygem-irb \
+        ruby-devel rubygem-irb \
         nodejs \
-        sudo which file shared-mime-info unzip jq git \
-        postgresql libpq-devel mysql-devel zlib-devel gd-devel libxml2-devel libxslt-devel \
-        make automake gcc gcc-c++ redhat-rpm-config \
+        sudo which file shared-mime-info jq git \
+        postgresql libpq-devel mariadb-devel mysql zlib-devel gd-devel libxml2-devel libxslt-devel \
```

Comment:
I believe we should use mysql library as in production. For now it is fine but has to be fixed. Otherwise we risk of missing some obscure issue.